tests/behat/features/media.feature

```diff
@@ -0,0 +1,8 @@
+@d7
+Feature: Check that MediaTrait works
+
+  @api
+  Scenario: Assert that user can attach file to media field
+    When I am logged in as a user with the "administrator" role
+    And I go to "node/add/article"
+    Then I attach the file "example_image.png" to "Image" media field
```

Comment:
this needs to save a page and assert that the file was indeed attached - you can do the assertion by css - check that image tag is present in a field. you may need to alter some field settings in the fixture drupal site.